config.yaml

```diff
@@ -1,9 +1,12 @@
-method: cheqd
-listener: 0.0.0.0:1313
-path: "/1.0/identifiers/:did"
-ledgerTimeout: 5s
-loglevel: debug
+ledger:
+  networks: "mainnet=rpc.cheqd.net:443;testnet=159.89.208.88:443"
```

Comment:
grpc.seed1.ap.cheqd.net will not work for the following reasons: 

1. We don't expose 9090
2. We route all requests via Cloudflare, which acts as a proxy. Essentially what this means is you can have subdomains only ONE subdomain level up, e.g., rpc.cheqd.net whereas eu.rpc.cheqd.net won't work since that's two levels. (They require a paid account to do two levels up.)

There are some nuances here specific to Cloudflare, so I can explain those.